control-agent/text-to-speech/movearm.py

```diff
@@ -0,0 +1,26 @@
+import stretch_body.robot
+import sys
+import time
+
+robot = stretch_body.robot.Robot()
+robot.startup()
+
+
+if sys.argv[3] == "in":
```

Comment:
I'd move sys.argv[3] to a variable, and then check if the code is running on main to make it more reusable

Reply:
done for all sys args